superset/datasets/commands/importers/v1/utils.py

```diff
@@ -137,7 +139,7 @@ def load_data(
             df[column_name] = pd.to_datetime(df[column_name])
 
     # reuse session when loading data if possible, to make import atomic
-    if example_database.sqlalchemy_uri == get_example_database().sqlalchemy_uri:
+    if example_database.sqlalchemy_uri == conf.get("SQLALCHEMY_DATABASE_URI"):
```

Comment:
I think you should check if they're the same OR the examples URI is null, as the loader logic uses the main URI as a fallback

Reply:
`example_database.sqlalchemy_uri` will be set to `SQLALCHEMY_DATABASE_URI` if `SQLALCHEMY_EXAMPLES_URI` is not set, so this should catch it, no?